.github/workflows/build-plugin-zip.yml

```
@@ -56,7 +56,7 @@ jobs:
                 github.event.inputs.version == 'rc' ||
                 github.event.inputs.version == 'stable'
               ) || (
-                endsWith( github.ref, needs.compute-stable-branches.outputs.current_stable_branch ) &&
+                github.ref != 'refs/heads/trunk' &&
```

Ideally, we should also prevent running the action from arbitrary branches 🤔 
Maybe we can limit to `release/` branches?

```suggestion
                startsWith( github.ref, 'refs/heads/release/' ) &&
```

---

Sounds good 👍🏻